### ISSUE TITLE
ability to filter over file path + filename

### ISSUE THREAD

**Issue by CommanderAsdasd** [jthaman/consult-everything#6]
Hi, thank for your recent fix, great tool available again.
But previously it was somewhat mightier because it was able to filter against file path, now it matches only against the filename/last entry after /. It was convenient be able to filter out certain path.
![image](https://github.com/user-attachments/assets/e6b3bcdb-1b67-4dc0-bc48-631c33e2070b)
![image](https://github.com/user-attachments/assets/b0c4c5a9-64b2-482c-858e-7ebd985f8cbb)
it matches searched substring in path but does no filtering based on it
![image](https://github.com/user-attachments/assets/904ceb35-249b-449a-8f7a-ded7fa1df725)
and the result will appear in the search only if the last part contains all occurrences of the string
![image](https://github.com/user-attachments/assets/7755098a-6ab9-476d-b89a-0f217f6cee1b)

How could I get that behavior back?

I have `completion-styles (orderless) `only.
GNU Emacs 30.0.50 (build 1, x86_64-w64-mingw32)

**Comment by jthaman** [comment]
Yes, i think that is a bug. Thanks for bringing it to my attention.
